### DISCUSSION TITLE
paperless update script seems broken

### DISCUSSION THREAD

**Discussion by yuriw** [tteck/Proxmox#2945] [Help Needed] [answered]
After running the `update` to upgrade from `2.7.2` to `2.8.2` error:

cat: /opt/Paperless-ngx_version.txt: No such file or directory
 ✓ Stopped all Paperless-ngx Services
 | Updating to v2.8.2   
[ERROR] in line 83: exit code 0: while executing command /usr/bin/python3 manage.py migrate &> /dev/null

Any clues on how to fix it are appreciated.

**Comment by tteck** [comment, marked as answer]
https://github.com/tteck/Proxmox/issues/2925
